pyro/infer/mcmc/mcmc.py

```diff
@@ -190,10 +190,11 @@ def _gen_samples(self, num_samples, init_params):
     # and transforms needed to do this wrapping. Note that only unconstrained parameters
     # are passed to `MCMCKernel` classes.
     def _trace_wrap(self, z, *args, **kwargs):
+        if self.kernel.transforms:
```

Comment:
Should we also address the other issue of `init_params` being respected with `models` in this PR?

Reply:
Yup, I just add a quick fix for it.